src/Kliva/Controls/SidePaneControl.xaml

```diff
@@ -23,7 +23,6 @@
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <ListView x:Name="TopMenu"
-                  SelectionChanged="OnTopMenuSelectionChanged"
```

Comment:
These handlers shouldn't be necessary anymore because the new code in SidePaneViewModel will enforce mutual exclusion between selection in the two lists.